src/helpers/sendFirebaseNotification.ts

```diff
@@ -0,0 +1,36 @@
+import { Message, getMessaging } from 'firebase-admin/messaging'
+import firebase from '@/helpers/firebase'
+
+const messaging = getMessaging(firebase)
+
+export default function (token: string, title?: string, postId?: number) {
+  const message = {
+    token,
+    android: {
+      priority: 'high',
+    },
+    data: {
+      postId: String(postId),
```

Comment:
If no `title` exists in `runNotificationListener`, `postId` will be equal to `undefined`. And on this line, you are converting `undefined` to string. Is it correct?
Should we create a data const equal to an empty object if `postId === undefined` and `{ postId: String(postId) }` if postId exists?